package-system/freetype/freetype.patch

```diff
@@ -0,0 +1,28 @@
+diff --git a/CMakeLists.txt b/CMakeLists.txt
+index 3ed55aad7..87a302d85 100644
+--- a/CMakeLists.txt
++++ b/CMakeLists.txt
+@@ -405,7 +405,7 @@ if (WIN32)
+ endif ()
+ 
+ if (BUILD_SHARED_LIBS)
+-  set_target_properties(freetype PROPERTIES
++set_target_properties(freetype PROPERTIES
+     VERSION ${LIBRARY_VERSION}
+     SOVERSION ${LIBRARY_SOVERSION})
+ endif ()
+diff --git a/include/ft2build.h b/include/ft2build.h
```

Comment:
All this part of the patch does is remove a space.
We can the patch portion for the CMakeLists.txt from the patch